### DISCUSSION TITLE
Compatibility with DX200 Lincoln ArcLinkXT interface

### DISCUSSION THREAD

**Discussion by ted-miller** [Yaskawa-Global/motoros2#116] [General]
Context: https://github.com/Yaskawa-Global/motoros2/pull/49#issuecomment-1668578286

> I reloaded my Lincoln M+ program alongside motoROS2 and received the following fatal errors from MotoROS2.
> ALARM 8012 MotoROS2 OUT OF MEMORY [1]

@acbuynak, can you tell me the version of your Lincoln interface? I removed all dynamic memory allocation back in 2016 (v2.8.1). So this shouldn't be an issue on newer versions of software.

`System Info` > `Version`. Then press `{PAGE}` key a couple times.

**Comment by acbuynak** [comment]
> Can you tell me the version of your Lincoln interface? 

My current M+ applications:
1. Lincoln ArcLinkXT `v3.1.0`
2. MotoROS2 `0.1.1pre`

**Reply by ted-miller** [reply]
Although I don't ever expect the DX2 Lincoln interface to be compatible with ROS, I am curious why there's a memory issue.
